pkg/controller/resourcequota/resource_quota_controller.go

```diff
@@ -228,18 +236,24 @@ func (rq *ResourceQuotaController) worker(queue workqueue.RateLimitingInterface)
 // Run begins quota controller using the specified number of workers
 func (rq *ResourceQuotaController) Run(workers int, stopCh <-chan struct{}) {
 	defer utilruntime.HandleCrash()
-	go rq.rqController.Run(stopCh)
+
+	glog.Infof("Starting resource quota controller")
+
 	// the controllers that replenish other resources to respond rapidly to state changes
 	for _, replenishmentController := range rq.replenishmentControllers {
 		go replenishmentController.Run(stopCh)
 	}
+
+	if !cache.WaitForCacheSync(stopCh, rq.informerSyncedFuncs...) {
```

Comment:
Yes, it's common for all controllers, and the wait is infinite until stopCh is closed (process terminating). Let's do it in a follow-up.